src/dioptra/restapi/v1/schemas.py

```diff
@@ -94,15 +141,15 @@ class BasePageSchema(Schema):
         attribute="is_complete",
         metadata=dict(description="Boolean indicating if more data is available."),
     )
-    first = fields.String(
+    first = fields.Url(
```

Comment:
This change is being handled in #393, so there will be a merge conflict to address.

Reply:
merged #393 to `dev` and rebased this branch